src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/FieldToken.cs

```diff
@@ -14,9 +14,9 @@ public struct FieldToken
     {
         public static readonly FieldToken Empty = default;
 
-        private readonly object _class;
+        private readonly object? _class;
 
-        internal FieldToken(int fieldToken, Type fieldClass)
+        internal FieldToken(int fieldToken, Type? fieldClass)
```

Comment:
this can be made non-nullable when the FieldBuilder comment is addressed.

Reply:
Changed.